tests/Aql/FilterExecutorTest.cpp

```diff
@@ -387,23 +387,23 @@ TEST_F(FilterExecutorTest, test_produce_datarange_need_more) {
   SharedAqlItemBlockPtr inBlock =
       buildBlock<1>(itemBlockManager,
                     {{R"(true)"}, {R"(false)"}, {R"(true)"}, {R"(false)"}, {R"(true)"}});
-
+  size_t hardLimit = 1000;
```

Comment:
Maybe keep that 1000 in a more "organized" way? Right now it is not super obvious.

Reply:
Ah, ok. It's just a test, nevermind.